devicetypes/smartthings/zigbee-multi-switch.src/zigbee-multi-switch.groovy

```diff
@@ -120,7 +125,7 @@ def parse(String description) {
 private void createChildDevices() {
 	def x = getChildCount()
 	for (i in 2..x) {
-		addChildDevice("Child Switch Health", "${device.deviceNetworkId}:0${i}", device.hubId,
+		addChildDevice("Child Switch Health", "${device.deviceNetworkId}:${i}", device.hubId,
```

Comment:
I think the leading 0 was here because the endpoint is hex and so we wanted it to be fixed length, similar to how the DNI is hex and includes leading 0s. I'd leave it as is unless there is a strong reason to change.

Reply:
if it's hex, the function that was parsing it back was treating it as decimal so there wasn't any consistency there so I did this for consistency's sake